openmc/data/njoy.py

```diff
@@ -147,6 +130,8 @@ def run(commands, tapein, tapeout, stdout=False, njoy_exec='njoy'):
         Dictionary mapping tape numbers to paths for any input files
     tapeout : dict
         Dictionary mapping tape numbers to paths for any output files
+    print_commands : bool, optional
```

Comment:
I think it would be more useful to have the inputs saved to a file. Why don't we have an argument `run(..., input_filename=None, ...)` that if it's set to a string will save the inputs to a file with that name. So below, you'd have something like:
```Python
if input_filename is not None:
    with open(input_filename, 'w') as input:
        input.write(commands)
```